pkg/compiler/codegen.go

```diff
@@ -2087,6 +2096,18 @@ func (c *codegen) writeJumps(b []byte) ([]byte, error) {
 			if op != opcode.PUSHA && math.MinInt8 <= offset && offset <= math.MaxInt8 {
 				offsets = append(offsets, ctx.IP())
 			}
+		case opcode.INITSLOT:
+			nextIP := ctx.NextIP()
+			locCount := c.reverseOffsetMap[ctx.IP()]
+			if argCount := b[nextIP-1]; locCount == 0 && argCount == 0 {
+				offsets = append(offsets, ctx.IP())
+				continue
+			}
+
+			if locCount > 255 {
+				return nil, errors.New("maximum of 255 local variables is allowed")
```

Comment:
So how can I find out which function is wrong?

Reply:
Fixed.